projects/packages/my-jetpack/_inc/components/product-cards-section/protect-card/use-protect-tooltip-copy.ts

```diff
@@ -0,0 +1,69 @@
+import { __, sprintf } from '@wordpress/i18n';
+import useProduct from '../../../data/products/use-product';
+import type { ReactElement } from 'react';
+
+type TooltipType = 'pluginsThemesTooltip' | 'scanThreatsTooltip';
+export type TooltipContent = {
+	[ key in TooltipType ]: {
+		title: ReactElement | string;
+		text: ReactElement | string;
+	};
+};
+
+/**
+ * Gets the translated tooltip copy based on Protect Scan details.
+ *
+ * @param {object} props - React props
+ * @param {number} props.pluginsCount - The number of installed plugins on the site.
+ * @param {number} props.themesCount - The number of installed themes on the site.
+ * @param {number} props.numThreats - The number of detected scan threats on the site.
+ * @returns {TooltipContent} An object containing each tooltip's title and text content.
+ */
+export function useProtectTooltipCopy( {
+	pluginsCount = 0,
+	themesCount = 0,
+	numThreats = 0,
+}: {
+	pluginsCount: number;
+	themesCount: number;
+	numThreats: number;
+} ): TooltipContent {
+	const slug = 'protect';
+	const { detail } = useProduct( slug );
+	const { hasPaidPlanForProduct: hasProtectPaidPlan } = detail;
+
+	return {
+		pluginsThemesTooltip: {
+			title: __( 'Improve site safety: secure plugins & themes', 'jetpack-my-jetpack' ),
+			text: sprintf(
+				/* translators: %1$d is the number of plugins and %2$d is the number of themes installed on the site. */
+				__(
+					'Your site has %1$d plugins and %2$d themes lacking security measures. Improve your site’s safety by adding protection at no cost.',
+					'jetpack-my-jetpack'
+				),
+				pluginsCount,
+				themesCount
+			),
+		},
+		scanThreatsTooltip:
+			hasProtectPaidPlan && numThreats
```

Comment:
I know this isn't going to be implemented until the next PR, but we'll want a case for a user that does not have the protect paid plan but does have threats. Right now if that were the case, the tooltip would say that all appears well even though they have threats

Reply:
Ah yes thanks. Yeah, this will implemented in the next PR. 👍